src/grt/grt-wave_opt_file-parse.adb

```diff
@@ -59,28 +60,28 @@ package body Grt.Wave_Opt_File.Parse is
                         return Boolean;
 
    -- Parse the line where the version is set
-   procedure Parse_Version (Line : String_Access);
+   procedure Parse_Version (Line : String; Line_Pos : Positive);
```

Comment:
So I will go for Lineno and Column (and they even have the same number 
of characters :) ).

I will fix that next time I have some changes to do in this file.


Le 28. 07. 16 à 07:43, tgingold a écrit :
>
> In src/grt/grt-wave_opt_file-parse.adb 
> <https://github.com/tgingold/ghdl/pull/124#discussion_r72566946>:
>
> > @@ -59,28 +60,28 @@ package body Grt.Wave_Opt_File.Parse is
> >                          return Boolean;
> >
> >     -- Parse the line where the version is set
> > -   procedure Parse_Version (Line : String_Access);
> > +   procedure Parse_Version (Line : String; Line_Pos : Positive);
>
> Col or Column is fine with me.
> The 'Line' meaning is to overloaded: is it the line string or the line 
> number ? But Col is clear.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/tgingold/ghdl/pull/124/files/eae288207d481e422e395aa7d310fa629d20432b#r72566946>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ATNePtJiDRmnSBBcptiPDMBVXzUiM390ks5qaEFlgaJpZM4JVgTD>.
>